nixos/doc/manual/release-notes/rl-2103.xml

```diff
@@ -344,6 +344,23 @@
      and rebooting.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     PulseAudio was upgraded to 14.0, with changes to the handling of default sinks.
+     See its <link xlink:href="https://www.freedesktop.org/wiki/Software/PulseAudio/Notes/14.0/">release notes</link>.
+    </para>
+
+    <para>
+     GNOME users may wish to delete their <literal>~/.config/pulse</literal> due to the changes to stream routing
+     logic. See <link xlink:href="https://gitlab.freedesktop.org/pulseaudio/pulseaudio/-/issues/832">PulseAudio bug 832</link>
+     for more information.
+    </para>
+
+    <para>
+     Flat volumes are now disabled by default. They can be reenabled by adding <literal>flat-volumes = yes</literal>
```

Comment:
Actually they were already disabled by default since dc923b6ad15dd60f4d18e5a8e3f7f211bc0246ff ?

Reply:
Aha, you're right. Yeah, I'll just delete this paragraph.

Reply:
but the corresponding line in the nixos module can be dropped, I think

Reply:
I'm going to leave it in for the moment to avoid surprises, and keep this a pure package upgrade.